fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationStoreDomain.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2015 Zalando SE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.zalando.stups.fullstop.violation;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * @author mrandi
+ */
+public class ViolationStoreDomain {
```

Comment:
So you will change it to 'Violation'? It's simpler to talk about 'Violation's with non-technical colleagues.